go/client/tree.go

```diff
@@ -21,6 +25,77 @@ type FileTree struct {
 	Dirs  map[string]*FileTree
 }
 
+func shouldIgnore(is *command.InputSpec, inp string, t command.InputType) bool {
+	for _, r := range is.InputExclusions {
+		if r.Type != command.UnspecifiedInputType && r.Type != t {
+			continue
+		}
+		if m, _ := regexp.MatchString(r.Regex, inp); m {
+			return true
+		}
+	}
+	return false
+}
+
+func collectFiles(execRoot string, is *command.InputSpec, path string, fs map[string][]byte) error {
+	absPath := filepath.Join(execRoot, path)
+	fi, err := os.Stat(absPath)
+	if err != nil {
+		return err
+	}
+	var t command.InputType
+	switch mode := fi.Mode(); {
+	case mode.IsDir():
+		t = command.DirectoryInputType
+	case mode.IsRegular():
+		t = command.FileInputType
+	default:
+		return fmt.Errorf("Unsupported input type: %s, %v", absPath, mode)
```

Comment:
errors should start with lowercase

Reply:
Right! I'm surprised go lint did not tell me this. The internal one does.